### ISSUE TITLE
Borderless fullscreen mode doesn't respect monitor selection

### ISSUE THREAD

**Issue by Sheepyhead** [bevyengine/bevy#5889]
## Bevy version

0.8

## What you did

```rs
app.insert_resource(WindowDescriptor {
            mode: WindowMode::BorderlessFullscreen,
            position: WindowPosition::Centered(MonitorSelection::Number(0)),
            ..Default::default()
        })
```

## What went wrong

I expected the game to full screen in monitor number 0, but it instead full screens on the primary monitor. It picks the correct screen when in windowed mode

## Additional information

On Windows 10 if it matters

**Comment by Sheepyhead** [comment]
Fullscreen and SizedFullscreen behave similarly

**Comment by devil-ira** [comment]
Duplicate of #5875. Will be fixed by #5878 :)

**Comment by rparrett** [comment]
#5878 was merged so I'll close this up.
